src/helpers/fog.js

```diff
@@ -1,39 +1,34 @@
-'use strict';
-
-var syllable = require('syllable');
+import { syllable } from 'syllable';
 
 function CalculateFogIndex(text) {
     // Clean up the text and create a word array from it
-    var textClean = text.replace(/[^a-zA-Z ]/g, "");
-    var wordArray = textClean.split(" ");
-    var wordArrayNoSpaces = wordArray.filter(v=>v!='');
-    var wordCount = wordArrayNoSpaces.length;
+    const textClean = text.replace(/[^a-zA-Z ]/g, "");
+    const wordArray = textClean.split(" ");
+    const wordArrayNoSpaces = wordArray.filter(v=>v!='');
+    const wordCount = wordArrayNoSpaces.length;
 
     // Figure out how many polysyllabic words are in the text
-    var polysyllabicWords = [];
-    for(var i=0; i < wordArray.length; i++) {
+    const polysyllabicWords = [];
```

Comment:
**nitpick**: I don't think this should be declared as a `const` given we're pushing values to it in the next code block

Reply:
My understanding of arrays is they can be const as the contents of the array can be changed.

For example:
```
// won't work
const polysyllabicWords = [];
polysyllabicWords = 'string';

// will work
const polysyllabicWords = [];
polysyllabicWords.push('string');
```

Reply:
Aha, this was my (lack of) knowledge of javascript conventions letting me down :) I've read the link you shared, and some other discussion (e.g. https://stackoverflow.com/questions/46553062/what-is-the-preferred-declaration-convention-for-objects-or-arrays-const-or-let) and understand this better now - thanks for explaining! No change required.